draft-packages/stories/Button.stories.tsx

```diff
@@ -17,6 +17,8 @@ export default {
   },
 }
 
+const clickAction = () => alert("This shouldn't fire")
```

Comment:
Surprised that we don't have any tests for Button. This would be a good unit test to have.